datasets/dandiarchive.yml

```diff
@@ -0,0 +1,29 @@
+Name: DANDI
+Description: >
+  DANDI is an public archive of neurophysiology datasets, including raw and processed data, 
+  and associated software containers. Datasets are shared according to a Creative Commons 
+  CC0 license. The data archive provides a broad range of cellular neurophysiology data. 
+  This includes electrode recordings, associated imaging and transcriptomic data using a 
+  set of community standards: [NWB:N - NWB:Neurophysiology](https://www.nwb.org/nwb-neurophysiology/),
+  [BIDS - Brain Imaging Data Structure](https://bids.neuroimaging.io/), and 
+  [NIDM - Neuro Imaging Data Model](http://nidm.nidash.org/). Development of DANDI is 
+  supported by the National Institute of Mental Health.
+Documentation: http://dandiarchive.org
+Contact: Support form at https://dandiarchive.org
+UpdateFrequency: New datasets deposited every 4-6 days
+Tags:
+  - aws-pds
+  - biology
+  - imaging
+  - neurophysiology
+  - BRAIN Initiative
+License: CC0
+Resources:
+  - Description: All datasets
+    ARN: arn:aws:s3:::dandiarchive
+    Region: us-east-2
+    Type: S3 Bucket
+DataAtWork:
+  Tutorials:
+  Tools & Applications:
```

Comment:
dandi-cli, dandiarchive.org ?

Reply:
we will populate these as things come along.